### PR TITLE
Inputing & evolving four principles

### PR DESCRIPTION
**Pull request by quaid**
Putting in Nick's original four principles from https://www.operate-first.cloud/blog/operate-first-operate-open-governance-and-hybrid and then bringing in some changes in further commits to this branch.

### CODE REVIEW

File: four-principles.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Operate First regroups a few concepts, relying on standard definitions of Open, Open Source, and Open Work:
+
+OPERATE:
+We build software optimized to run as a service and allow others to run it as a service easily, including documenting how we operate the cloud environment itself.
+
+OPEN: We develop open source practices, tooling, and documentation that allows us to build and manage our software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
```
Comment (quaid) [inline four-principles.md:6]: Suggestion offered and accepted from @chirino that this be changes so it's clear we mean everyone, everywhere:
```suggestion
OPEN: We develop open source practices, tooling, and documentation that allows anyone to build and manage our software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
```

File: four-principles.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Operate First regroups a few concepts, relying on standard definitions of Open, Open Source, and Open Work:
+
+OPERATE:
+We build software optimized to run as a service and allow others to run it as a service easily, including documenting how we operate the cloud environment itself.
```
Comment (quaid) [inline four-principles.md:4]: Seeing how to make this feel inclusive of the reader/anyone.

```suggestion
To build software optimized to run as a service and allow others to run it as a service easily, including documenting how to operate the cloud environment itself.
```

File: four-principles.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+OPEN: We develop open source practices, tooling, and documentation that allows anyone to build and manage our software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
```
Comment (quaid) [inline four-principles.md:6]:
```suggestion
OPEN: To develop open source practices, tooling, and documentation that allows anyone to build and manage the software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
```

File: four-principles.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+GOVERNANCE: We foster inclusive, respectful, and collaborative communities through clear standards and accepted practices (e.g., codes of conduct) so participants are empowered to contribute.
```
Comment (quaid) [inline four-principles.md:8]:
```suggestion
GOVERNANCE: To foster inclusive, respectful, and collaborative communities through clear standards and accepted practices (e.g., codes of conduct) so participants are empowered to contribute.
```

File: four-principles.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+HYBRID: We ensure that there is no dependency in our tooling to the cloud environment we are operating in by always verifying our software and management tooling works identically on at least one other platform (another cloud, in a private data center, …)
```
Comment (quaid) [inline four-principles.md:10]:
```suggestion
HYBRID: To ensure there is no dependency in the tooling of the cloud environment operating in by always verifying the software and management tooling works identically on at least one other platform (another cloud, in a private data center, …)
```

**Comment by nijaba** [comment]
These changes make a lot of sense.  +1 for me.  Thanks

File: four-principles.md (hunk 3 above)
Comment (billburnseh) [reply four-principles.md:6]: /lgtm

File: four-principles.md (hunk 4 above)
Comment (billburnseh) [reply four-principles.md:8]: /lgtm

File: four-principles.md (hunk 5 above)
Comment (billburnseh) [reply four-principles.md:10]: Sentence reads a little awkardly to me. Suggest removing the "operating in" or adding more to it, like "...the software is operating in"

**Comment by billburnseh** [comment]
approved

**Comment by emmanuelbernard** [comment]
@billburnseh I think something went wrong. These changes do not show up in https://github.com/operate-first/community/commit/6a9ea45dd33a2d00397443713aaaaba55ab9604b nor in https://github.com/operate-first/community/blob/main/four-principles.md

**Comment by billburnseh** [comment]
Karsten made me owner, so not sure how to finish this as I was the reviewer, Will try and pick a reviewer to get it unstuck.

**Comment by billburnseh** [comment]
Humair, looking for a reviewer....

**Comment by quaid** [comment]
> /lgtm
> 

What is the link to the comment with changes you made that isn't merged into my branch?

**Comment by quaid** [comment]
> > /lgtm
> > 
> 
> What is the link to the comment with changes you made that isn't merged into my branch?

@billburnseh it looks like we never committed the suggestion(s) before the PR was merged.

Can you instead open a new PR against the committed file? And then make all the changes in that patch?

You'll need to fork the repo to gihub.com/billburnseh, make the changes there, and then return to the main repo and the pull request dialog should be there. Then it's a clean PR instead of reverting #220 to commit the suggestions. (Or figure out how to get them from the merged branch into a live branch ...)

**Pull request closed without merging**